activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+*   Disable validations
```

Comment:
Hwo about renaming this (and the PR) to something like
```suggestion
*   Allow disabling validations per instance
```

Reply:
Great suggestion! Thank you!